app/infrastructure/services.py

```diff
@@ -15,6 +15,7 @@ def search(self, filters: dict) -> Tuple[List[Organization], int, int]:
         page = filters.pop('page')
         page_size = filters.pop('page_size')
         search_text = filters.pop('q')
+        sort = filters.pop('sort', None)
```

Comment:
It currently doesn't work for all sorts defined in https://github.com/opendatateam/udata/blob/af594eafdd590386d391f5258be25d42c0893987/udata/core/dataset/search.py#L22. (ex `created_at` instead of `created`).